.travis.yml

```diff
@@ -9,7 +9,7 @@ install:
   - pip install coveralls
 
 script:
-  - coverage run --source=authors/ manage.py test
+  - coverage run --source=authors/ --omit=authors/wsgi.py,authors/settings/production.py,authors/settings/staging.py manage.py test
```

Comment:
I think these files can be excluded with the exclude tag. In cases where there are so many files to exclude, you wouldn't need to chain them in that one command

Reply:
I resolved it @reiosantos